app/helpers/application_helper.rb

```diff
@@ -33,7 +32,7 @@ def link_to_runner(runner_url, form_id, form_slug, mode: :preview_draft)
   end
 
   def contact_url
-    "mailto:govuk-forms-support@govuk.zendesk.com"
+    t("contact_url")
```

Comment:
I don't think we can change this here - it's used in the `contact_link` helper below. That's used in other places (like [the 403 page](https://github.com/alphagov/forms-admin/blob/2ef3760b74fca7ddfd252c414f88462b0fa09191/app/views/errors/forbidden.html.erb#L6)) where I think we still want the email link rather than this survey

Reply:
Ah interesting! OK thanks I'll rework it

Reply:
The refactor looks good overall, but I think to update the link in the phase banner we'll need to change it somewhere more specific

Reply:
Update looks good!